test_scripts/API/VehicleData/StabilityControlsStatus/001_GetVehicleData_StabilityControlsStatus.lua

```diff
@@ -0,0 +1,37 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: Check receiving StabilityControlsStatus data via GetVehicleData RPC
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed
+-- 4) App is activated
+
```

Comment:
```suggestion
--
```
I would propose to comment out this line and in the other tests below

Reply:
@VjKlepikov Fixed (see a7a20fe)